pad-tls/pad-tls.c

```diff
@@ -0,0 +1,112 @@
+/* pad-tls: tool to pad TLS segment of ELF files so that a given compartment's
```

Comment:
Nit: for tools like these, C++ or Rust might make your life easier (although C works fine too).